src/Security/Security.php

```diff
@@ -884,7 +884,7 @@ function (RequestHandler $handler) {
      * @param RequestHandler $handler
      * @param string $title The title of the form
      * @param array $templates
-     * @return array|HTTPResponse|RequestHandler|DBHTMLText|string
+     * @return HTTPResponse|RequestHandler|DBHTMLText|string
      */
     protected function delegateToHandler(RequestHandler $handler, $title, array $templates = [])
```

Comment:
This and `delegateToMultipleHandlers()` explicitly _cannot_ return an array, because they transform arrays with a call to `renderWrappedController()`